pkg/apis/mysql/v1/cluster.go

```diff
@@ -95,6 +95,11 @@ type MySQLClusterSpec struct {
 	// ConfigRef allows a user to specify a custom configuration file for MySQL.
 	// +optional
 	ConfigRef *corev1.LocalObjectReference `json:"configRef,omitempty"`
+
+	// SSLSecretRef allows a user to specify custom CA certificate, server certificate
+	// and server key for group replication ssl
+	// +optional
+	SSLSecretRef *corev1.LocalObjectReference `json:"sslSecretRef,omitempty"`
```

Comment:
Maybe it's bikeshedding but I'm not sure we made the right decision re `{Foo}Ref` I'd rather `SSLSecret`, `RootPasswordSecret` and `MyCNF` or similar. Probably out of scope though.

Reply:
This should be a separate issue to do them all at once

Reply:
Agree with you both. I'm not a huge fan. Let's do this afterwards.